src/components /NotFound.jsx

```diff
@@ -0,0 +1,8 @@
+export default function NotFound() {
+  return (
+    <div style={{ textAlign: "center", marginTop: "50px" }}>
+      <h1>404 - Not Found</h1>
+      <p>The page you are looking for does not exist.</p>
+    </div>
+  );
```

Comment:
When I put in a non-existent article id, like "/SingleArticle/1222", this element appears for a second before redirecting to "/NotFound" which is blank. The same behaviour is happening for an invalid article id 
This might be due to an error occuring with an axios request (which you can see in the console)
<img width="1347" alt="image" src="https://github.com/Geno419/Reddit-Clone-Fe/assets/151947206/c031f3c7-87c3-48d4-8d06-b406c36acd9a">